packages/manager/modules/common-api/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@ovh-ux/manager-module-common-api",
+  "version": "0.1.0",
+  "private": true,
+  "description": "OVHcloud manager module common API (order, services, me, ...).",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ovh/manager.git",
+    "directory": "packages/manager/modules/common-api"
+  },
+  "license": "BSD-3-Clause",
+  "author": "OVH SAS",
+  "sideEffects": false,
+  "main": "./dist/index.js",
+  "module": "./dist/index.js",
+  "types": "./dist/types/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "tsc",
+    "dev": "tsc",
+    "start:watch": "tsc -w",
+    "test": "vitest run",
+    "test:coverage": "vitest run --coverage"
+  },
+  "dependencies": {
+    "@ovh-ux/manager-core-api": "^0.9.0",
```

Comment:
You can pass manager-core-api and manager-shell-client in peerDependencies

Reply:
done thanks